packages/adapter-components/src/elements/ducktype/standalone_field_extractor.ts

```diff
@@ -131,6 +131,7 @@ const addFieldTypeAndInstances = async ({
         transformationConfigByType,
         transformationDefaultConfig,
         getElemIdFunc,
+        nestedPath: [type.elemID.name, inst.elemID.name, fieldName],
```

Comment:
this looks good :) but does this work recursively when there's a nested resource under a nested resource? I think we might need to base it more closely on the instance's path and not the type path? it'd be a little ugly though, something like
```suggestion
nestedPath: [
          ...inst.path?.slice(2, inst.path?.length - 1) ?? [],
          fieldName,
        ],
```
which is not ideal.
(this is still correct for the "basic" most common case, so can also merge and figure it out later)

Reply:
Great :)